packages/loader/container-loader/src/container.ts

```diff
@@ -1317,7 +1321,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this._protocolHandler =
             await this.loadAndInitializeProtocolState(attributes, undefined, snapshotTree);
 
-        await this.createDetachedContext(attributes, snapshotTree);
+        const codeDetails = this.getCodeDetailsFromQuorum();
+        await this.instantiateContext(
+            codeDetails,
+            attributes,
+            true,
```

Comment:
I feel like we should either allow all methods within this class to use this._existing and assume it's correct, or start removing it. With current approach there is a chance that code would get unsynchronized (i.e. boolean here and assignment to this._existing above in this function would get out of sync).
Would be also great to add comments on what args mean, i.e.
```typescript
true, // existing
```

Reply:
Agree with the comment about the free-flow boolean in there. I don't believe I'm able to remove the property from the class at this stage :(